DemoApp/DemoApp/Views/TMModalPage.xaml

```diff
@@ -3,17 +3,18 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              x:Class="DemoApp.Views.TMModalPage"
              xmlns:Modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             xmlns:control="clr-namespace:Trimble.Modus.Components.Controls;assembly=Trimble.Modus.Components"
              Title="TMModalPage">
   <ScrollView>
     <VerticalStackLayout Spacing="5"
                          Padding="8">
       <Label Text="Modal"
              FontSize="Medium"
              FontAttributes="Bold" />
-      <Modus:TMInput x:Name="ModalTitle"

```

Comment:
The name `Modus` was obvious. But `control` doesn't actually convey what it means.

Reply:
Modified to modus